### PR TITLE
Ashkan/add blog&career sections

### PR DESCRIPTION
**Pull request by ashkanx**
- Add a section to home page for blog
image
- Add a new section to career page
image
- add partners & bulk trade pages to sitemap

### CODE REVIEW

File: src/templates/haml/home/index.html.haml [hunk 1]
```diff
@@ -45,31 +45,50 @@
...
+            %h3= l('Sharp prices')
+            %p= l('Binary.com\'s patented pricing technology allows you to benefit from the same rates of return as traders in the interbank market.')
+          %li.xlist2
+            %h3= l('Your funds are safe')
+            %p= l('With Binary.com, your money is secure. We segregate and hold client funds in secure AAA-rated banks.')
+      .grd-grid-6.grd-grid-mobile-12.grd-grid-phablet-12
+        %ul
+          %li.xlist3
+            %h3= l('Trade your way')
+            %p= l('Don\'t limit yourself! Whatever your market view, Binary.com allows you to put your strategy to work and profit from your predictions.')
```
Comment (raunakkathuria) [inline src/templates/haml/home/index.html.haml]: use double quotes here rather than escaping it

File: src/templates/haml/home/index.html.haml [hunk 2]
```diff
@@ -45,31 +45,50 @@
...
+            %p= l('Binary.com is the award-winning industry pioneer in online options trading.')
+#xacademy
+  .grd-container
+    .grd-row-padding
+      %h1.center-aligned.head-academy= l('Binary.com academy')
+      .grd-grid-12.grd-grid-mobile-12.grd-grid-phablet-6.center
+        %p!= l('Improve your trading skills with our free educational & training resources.')
+      .grd-grid-3.grd-grid-mobile-12.grd-grid-phablet-6
+      .grd-grid-3.grd-grid-mobile-12.grd-grid-phablet-6.center
+        %img.responsive{:alt => "", :src => url_for("images/pages/home/academy-icon.svg")}
```
Comment (raunakkathuria) [inline src/templates/haml/home/index.html.haml]: remove alt if its empty

File: src/templates/haml/home/index.html.haml (hunk 1 above)
Comment (ashkanx) [inline src/templates/haml/home/index.html.haml]: This is old one :)
I will change it.

Comment (ashkanx) [inline src/templates/haml/home/index.html.haml]: done

File: src/templates/haml/home/index.html.haml (hunk 2 above)
Comment (ashkanx) [inline src/templates/haml/home/index.html.haml]: done

**Pull request merged**